internal/cmpimg/cmpimg.go

```diff
@@ -0,0 +1,78 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// package cmpimg compares the raw representation of images taking into account idiosyncracies related to their underlying format (SVG, PDF, PNG, ...)
+package cmpimg
+
+import (
+	"bytes"
+	"image"
+	"reflect"
+
+	"rsc.io/pdf"
+)
+
+// Equal takes the raw representation of two images, raw1 and raw2,
+// together with the underlying image type ("pdf", "svg", ...), and returns
```

Comment:
Please fully enumerate the types that are accepted; we generate EPS images, and I don't see that those are handled here.